docs/app/pages/components/sections/panels/item-display-panel-inline/item-display-panel-inline.component.ts

```diff
@@ -0,0 +1,384 @@
+            extension: '.doc',
+            storage: chance.d100().toString(),
+            active: false,
+            panel: {
+                title: 'Site Detail - UX Aspects (DOC)',
+                content: this.modalDoc
+            }
+        }, {
+            id: 20,
+            name: chance.name(),
+            dateString: '3 July 2017',
+            document: 'Document 20.pdf',
+            extension: '.pdf',
+            storage: chance.d100().toString(),
+            active: false,
+            panel: {
+                title: 'Site Detail - UX Aspects (PDF)',
+                content: this.modalPdf
+            }
+        }
+    ];
+
+    noSelectedItem = `<div *ngIf="!selectedItem" class="preview-no-record">
+    <i class="hpe-icon hpe-document"></i>
+    <h2 ng-bind="previewEmptyText" class="ng-binding">Select an item from the list</h2>
+</div>`;
+
+    constructor(colorService: ColorService) {
+        super(require.context('./snippets/', false, /\.(html|css|js|ts)$/));
+
+        this.sparkTrackColor = colorService.getColor('accent').setAlpha(0.2).toRgba();
+        this.sparkBarColor = colorService.getColor('accent').toHex();
+    }
+
+    selectItem(item: Item) {
+        this.shadow = true;
+        this.selectedItem = item;
+        this.updatePanel();
+    }
+
+    togglePanel() {
+        this.visible = !this.visible;
+    }
+
+    previous() {
+        if (this.previousEnabled) {
+            let id = this.selectedItem.id - 1;
+            this.selectedItem = this.items[id - 1];
+            this.updatePanel();
+        }
+    }
+
+    next() {
+        if (this.nextEnabled) {
+            let id = this.selectedItem.id + 1;
+            this.selectedItem = this.items[id - 1];
+            this.updatePanel();
+        }
+    }
+
+    upArrow(event: KeyboardEvent) {
+        if (this.visible) {
+            event.preventDefault();
+            this.previous();
+        }
+    }
+
+    downArrow(event: KeyboardEvent) {
+        if (this.visible) {
+            event.preventDefault();
+            this.next();
+        }
+    }
+
+    updatePanel() {
+
+        if (this.selectedItem.id < 20) {
+            this.nextEnabled = true;
+        } else {
+            this.nextEnabled = false;
+        }
+
+        if (this.selectedItem.id > 1) {
+            this.previousEnabled = true;
+        } else {
+            this.previousEnabled = false;
+        }
+    }
+
+    public plunk: IPlunk = {
+        files: {
+            'app.component.ts': this.snippets.raw.appTs,
+            'app.component.html': this.snippets.raw.appHtml,
+            'app.component.css': this.snippets.raw.appCss
+        },
+        mappings: [
+            {
+                alias: 'chance',
+                source: 'npm:chance@1.0.6'
+            }
+        ],
+        modules: [{
+            imports: ['ItemDisplayPanelModule', 'ColorServiceModule', 'SparkModule'],
+            library: 'ux-aspects'
+        }]
+    };
+
+}
+
+interface Item {
```

Comment:
Might be good to name these something like `DisplayPanelItem` and `DisplayPanel` to ensure no future naming conflicts.